applications/grafana/chart/config/dashboards/datasource-checker.json

```diff
@@ -101,7 +101,7 @@
             "hide": false,
             "metricColumn": "none",
             "rawQuery": true,
-            "rawSql": "SELECT *\nFROM mttest.etairqualityobserved\n\n",
+            "rawSql": "SELECT *\nFROM etairqualityobserved\n\n",
```

Comment:
Per this other PR, https://github.com/FIWARE-Ops/marinera/pull/34 , is this still valid?

Reply:
This was because I made a rebase on that branch for the test to work. @wistefan could say more about this